recipes/org-re-reveal-citeproc

```diff
@@ -0,0 +1,4 @@
+(org-re-reveal-citeproc
+ :repo "oer/org-re-reveal-citeproc"
+ :fetcher gitlab
+ :files (:defaults "LICENSES" "README.org" "local.css" "references.bib"))
```

Comment:
MELPA doesn't package this information, unless you have a specific need for it that I'm missing:
```suggestion
 :files (:defaults "local.css" "references.bib"))
```